apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/BulkApiLiveTest.java

```diff
@@ -69,7 +69,7 @@ public void testExtractArchive() throws Exception {
                                                      .extractArchive(containerName, payload, "tar.gz");
          assertEquals(extractResponse.getCreated(), OBJECT_COUNT);
          assertTrue(extractResponse.getErrors().isEmpty());
-         assertEquals(api.getContainerApi(regionId).get(containerName).getObjectCount(), OBJECT_COUNT);
+         assertEquals(api.getContainerApi(regionId).get(containerName).getObjectCount(), Long.valueOf(OBJECT_COUNT));
```

Comment:
Is this needed (here and elsewhere)? If you let OBJECT_COUNT be autoboxed, the test spec will reflect the backwards compatibility of the change, as least in source, as was mentioned. Thoughts?